website/versioned_docs/version-3.0/examples/export_entire_dataset.mdx

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
the same example needs to go to `docs/examples` too, this is just a version snapshot